### PR TITLE
when camera_type is equirectangular, add spherical metadata to rendered video

### PR DESCRIPTION
**Pull request by machenmusik**
Implements #1149.

### CODE REVIEW

File: scripts/render.py [hunk 1]
```diff
@@ -42,6 +44,7 @@ def _render_trajectory_video(
     rendered_resolution_scaling_factor: float = 1.0,
     seconds: float = 5.0,
     output_format: Literal["images", "video"] = "video",
+    camera_type: str = "",
```
Comment (tancik) [inline scripts/render.py:47]: Can you make this a Literal and add it to the doc string below.

**Review by tancik — COMMENTED** [review]
Thanks for digging into the details for this functionality. It's a great addition!

File: scripts/render.py (hunk 1 above)
Comment (machenmusik) [reply scripts/render.py]: I discovered the CameraType enum and switched to that instead.

**Comment by machenmusik** [comment]
(Hang on, more changes incoming)

**Comment by machenmusik** [comment]
It seemed inefficient for all rendered images to be held in memory just to encode video, so that is no longer done.

As a curious side effect, I noticed that non-filename trajectories such as spiral don't have a defined render width or height.  That didn't matter before when passing all the images at once to be encoded,, but it does now that we create the video writer before we have the first image (and its size).  The values for Nerfacto are currently the default for spiral trajectory, but if there is a better way to know what resolution should be expected, that would be preferable.

For later perhaps: it should be possible to reuse camera path files, but override the width and height for a given render command.

**Comment by machenmusik** [comment]
(okay, finally got isort / black / pylint to agree...)

File: scripts/render.py [hunk 2]
```diff
@@ -53,9 +59,11 @@ def _render_trajectory_video(
         rendered_resolution_scaling_factor: Scaling factor to apply to the camera image resolution.
         seconds: Length of output video.
         output_format: How to save output data.
+        camera_type: Camera projection format type.
+        render_width: Video width to render.
```
Comment (tancik) [inline scripts/render.py:63]: Nit: doc order should match definition order

**Review by tancik — APPROVED** [review]
LGTM, this restructure looks good to me. Thanks for working on this.

File: scripts/render.py (hunk 2 above)
Comment (machenmusik) [reply scripts/render.py:63]: Must have been one of the isort / black / pylint fixes, forgot to update docs

**Comment by tancik** [comment]
Looks good, feel free to merge. You should now have access (check your email)

**Comment by machenmusik** [comment]
Done thx

**Pull request merged**